### PR TITLE
Fix 1-way audio bug

### PR DESCRIPTION
**Pull request by djoksimo**
Fixes the 1-way audio issue:

TL;DR: only one user can talk with the native webrtc adapter. This PR should fix this bug.

Closes https://github.com/networked-aframe/networked-aframe/issues/194

Adding this snippet seems to do the trick:

```javascript
             localStream.getTracks().forEach(
                track => {
                  Object.keys(self.peers).forEach(peerId => { 
                  self.peers[peerId].pc.addTrack(track, localStream) 
                })
              })
```

Safari doesn't seem to work (before & after this PR), but the [commented link](https://github.com/OpenVidu/openvidu/blob/master/openvidu-browser/src/OpenViduInternal/WebRtcPeer/WebRtcPeer.ts#L154) in the webrtc adapter file seems useful. I'll try to fix Safari functionality in this PR as well.

Tested by:

- opening two tabs in Chrome for localhost:8080/basic-audio.html.
- served over HTTPS using ngrok
- heard echo with earbuds

In 1-tab:

Already had audio for Is3UNSNjGAK2Jr4pAAAE in 1st tab.
Already had audio for E3Yb914FUJgxNX5pAAAD in 2nd tab.

I think that we can prevent duplicating redundant function calls:

e.g.:
> Already had ...

But this would require some exploration

### CODE REVIEW

**Comment by vincentfretin** [comment]
Not related to native webrtc adapter per se, I use on my project mozilla janus adapter and was having issue not hearing audio on Safari, I applied the hack that Hubs is doing for this issue, you can find the comment and code here: https://github.com/mozilla/hubs/blob/0cf532c509ca792438952414968cbe814f21b1b5/src/hub.js#L1426-L1470

File: src/adapters/naf-webrtc-adapter.js [hunk 1]
```diff
@@ -367,16 +367,22 @@ class WebrtcAdapter {
...
-            self.storeAudioStream(self.myId, localStream);
-            self.connectSuccess(self.myId);
-          })
-          .catch(e => {
-            NAF.log.error(e);
-            console.error("Microphone is disabled due to lack of permissions");
-            self.sendAudio = false;
-            self.connectSuccess(self.myId);
-          });
+            .then(localStream => {
```
Comment (osresearch) [inline src/adapters/naf-webrtc-adapter.js:370]: Is the white space change a spurious diff?

**Comment by osresearch** [comment]
With this patch applied this seems to work! Thanks for tracking down the issue.

Although testing it took me down a debugging path -- the webrtc demo doesn't request mic permissions; so none of the participants had any audio.

File: src/adapters/naf-webrtc-adapter.js (hunk 1 above)
Comment (djoksimo) [reply src/adapters/naf-webrtc-adapter.js:370]: I added this line as well:

```
 localStream.getTracks().forEach(
                track => {
                  Object.keys(self.peers).forEach(peerId => { 
                  self.peers[peerId].pc.addTrack(track, localStream) 
                })
              })
```

I'll revert the whitespace diff
